src/main/java/CalculatorTesting/TestTypes/PostfixToResultCases/DoubleOperator.java

```diff
@@ -0,0 +1,46 @@
+package CalculatorTesting.TestTypes.PostfixToResultCases;
+
+import ShuntingYardCalculator.Calculator.CalculationSteps.PostfixToResult;
+import ShuntingYardCalculator.Enums.Type;
+import javafx.util.Pair;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+
+import static CalculatorTesting.ParsePostfixString.parse;
+import static org.junit.Assert.assertThrows;
+
+public class DoubleOperator {
```

Comment:
It is okay  that you type one test in class but usually there are many tests for one, and the test name should indicate what the test does, it depends what language you are using, In Java you have a suffix of 'Tests' to you class name:
```
public class OpertorTests {
  @Test
  public void doubleOpertorThrowsInputMismatch {
    // Arrange
   code = code;

    // Act
    callMethod();

    // Assert
    this.assertEquals(actValue, expectedValue); // or in different order
}
```